lstm.py

```diff
@@ -85,22 +86,25 @@ def prepare_sequences(notes, n_vocab):
     return (network_input, network_output)
 
 def create_network(network_input, n_vocab):
-    """ create the structure of the neural network """
-    model = Sequential()
-    model.add(LSTM(
-        512,
-        input_shape=(network_input.shape[1], network_input.shape[2]),
-        return_sequences=True
-    ))
-    model.add(Dropout(0.3))
-    model.add(LSTM(512, return_sequences=True))
-    model.add(Dropout(0.3))
-    model.add(LSTM(512))
-    model.add(Dense(256))
-    model.add(Dropout(0.3))
-    model.add(Dense(n_vocab))
-    model.add(Activation('softmax'))
-    model.compile(loss='categorical_crossentropy', optimizer='rmsprop')
+    try:
+        model = load_model(sys.argv[2])
```

Comment:
I would like you to revert this change. The model of the repository should be the one in the article, for consistency. Also I'm going to do a minor update to the article and repository soon, which will improve the model slightly.